references/settings.md

```diff
@@ -4,14 +4,14 @@
 
 <RouteHighlighter method="GET" route="/indexes/:uid/settings" />
 
-Get the index settings.
+Get settings for a given index.
```

Comment:
I think it should be more straight to the point. Not sure 
```suggestion
Get settings of an index
```

Reply:
I choose this kind of sentence ('for a given index') because it's used all along our documentation ^^'
But I agree.